### PR TITLE
Add docs for Volume Management

### PR DESCRIPTION
**Pull request by usha-mandya**
Signed-off-by: Usha Mandya <usha.mandya@docker.com>

Added docs on how to Manage volumes using the Docker Dashboard

### CODE REVIEW

File: desktop/dashboard.md [hunk 1]
```diff
@@ -220,3 +224,21 @@ To interact with remote repositories:
     The **View in Hub** option opens the Docker Hub page and displays detailed information about the image, such as the OS architecture, size of the image, the date when the image was pushed, and a list of the image layers.
 
     If you have subscribed to a Pro or a Team plan and have enabled [Vulnerability Scanning](../docker-hub/vulnerability-scanning.md) the Docker Hub page also displays a summary of the vulnerability scan report and provides detailed information about the vulnerabilities identified.
+
+## Manage volumes
```
Comment (mat007) [inline desktop/dashboard.md:228]: The previous section is `Explore your images`, should this one be `Manage your volumes`?

File: desktop/dashboard.md [hunk 2]
```diff
@@ -220,3 +224,21 @@ To interact with remote repositories:
...
+
+The **Volumes** view in Docker Dashboard enables you to easily create and delete volumes and see which ones are being used. If you are a developer with a Pro or a Team subscription, you can also see which container is using a specific volume and explore the files and folders in your volumes.
```
Comment (mat007) [inline desktop/dashboard.md:230]: nit: should we link «a Pro or a Team subscription» to the pricing matrix or somewhere similar? I’m not sure if/how it’s done elsewhere

Comment (usha-mandya) [reply desktop/dashboard.md:230]: Good point. Let me add a note

Comment (usha-mandya) [reply desktop/dashboard.md]: Updated

File: desktop/dashboard.md (hunk 1 above)
Comment (usha-mandya) [reply desktop/dashboard.md:228]: Added "Manage Volumes" as a heading to be consistent with "Volume Management" in the blog draft.

Comment (mat007) [reply desktop/dashboard.md:228]: What I meant is the TOC looks a bit inconsistent

image

Comment (usha-mandya) [reply desktop/dashboard.md]: Flipped the order of the headings.

**Review by dieucao — APPROVED** [review]
LGTM

**Pull request merged**